test/keyboard-input.html

```diff
@@ -700,6 +700,66 @@
 
         });
 
+        describe('auto-open-disabled', () => {
+
+          beforeEach(() => {
+            datepicker.autoOpenDisabled = true;
+          });
+
+          it('should not open overlay on input', () => {
+            inputChar('j');
+            expect(datepicker.opened).not.to.be.true;
+          });
+
+          it('should set datepicker value on blur', () => {
+            inputChar('1/1/2000');
+            datepicker.dispatchEvent(new Event('blur'));
+            expect(datepicker.value).to.equal('2000-01-01');
+          });
+
+          it('should not be invalid on blur if valid date is entered', () => {
+            inputChar('1/1/2000');
+            datepicker.dispatchEvent(new Event('blur'));
+            expect(datepicker.invalid).not.to.be.true;
+          });
+
+          it('should revert input value on esc when overlay not initialized', () => {
+            inputChar('1/1/2000');
+            esc();
+            expect(datepicker._inputValue).to.equal('');
+            expect(datepicker.value).to.equal('');
+          });
+
+          it('should revert input value on esc when overlay has been initialized', () => {
+            datepicker.open();
+            datepicker.close();
+            inputChar('1/1/2000');
+            esc();
+            expect(datepicker._inputValue).to.equal('');
+            expect(datepicker.value).to.equal('');
+          });
+
+          it('should apply the input value on enter when overlay not initialized', () => {
+            inputChar('1/1/2000');
+            enter();
+            expect(datepicker.value).to.equal('2000-01-01');
+          });
+
+          it('should apply input value on enter when overlay has been initialized', () => {
```

Comment:
This test description looks invalid (the test is about reverting the value on esc)

Reply:
Ah sorry my mistake, fixed now